api-docs/client-interactions.md

```diff
@@ -88,14 +84,12 @@ GET /api/clients/:id/interactions/
     },
     {
       "id": 2,
-      "title": "Financial Counseling - Financial Coach Referal",
       "serviceId": 13,
       "interactionType": "inPerson",
       "description": "Appointment setup with financial coach. Client would like to save up enough for a down payment on a car better suited for their work.",
       "dateofInteraction": "2019-05-06",
       "duration": "1:00:00",
       "location": "CUOffice",
-      "canModify": true,
       "isDeleted": false,
```

Comment:
Should the isDeleted be removed here also?

Reply:
No. You don't send `isDeleted` up in requests, but it does come down in responses. This one is in a response.